tools/functional-tester/etcd-runner/main.go

```diff
@@ -212,6 +219,136 @@ func runRacer(eps []string, round int) {
 	doRounds(rcs, round)
 }
 
+func runWatcher(eps []string) {
+	runningTime := 60 * time.Second // time for which operation should be performed
+	noOfPrefixes := 36              // total number of prefixes which will be watched upon
+	watchPerPrefix := 10            // number of watchers per prefix
+	reqRate := 20                   // put request per second
```

Comment:
ok will check by putting some logs to ensure the same. If it works will update the PR and let you know.
Thanks!